support/doc/plugins/guide.md

```diff
@@ -323,7 +323,9 @@ async function register ({
       const streamString = streamNum ? ':' + streamNum : ''
 
       // You can also return a promise
+      // All these options are optional and defaults to []
```

Comment:
You should add the new `scaleFilter` to the documentation and specify its default value.